nrich-excel/src/main/java/net/croz/nrich/excel/converter/DefaultCellValueConverter.java

```diff
@@ -60,12 +61,14 @@ private ConverterHolder findConverter(Object value) {
                 .orElse(null);
     }
 
-    @Value
```

Comment:
We should use `@Value` (capital) in the commit message to be consistent with the code.

Reply:
This seems to be an issue with how it is shown here, in gitlog I see  `@Value`